packages/flutter/test/material/chip_test.dart

```diff
@@ -188,6 +188,93 @@ void _expectCheckmarkColor(Finder finder, Color color) {
   );
 }
 
+Future<void> pumpFrames({
+  WidgetTester tester,
+  int frames,
+  Duration duration,
+}) async {
+  int i;
+  for (i = 0; i < frames; i ++) {
+    await tester.pump(duration);
+  }
+}
+
+Widget _chipWithOptionalDeleteButton({
+  UniqueKey deleteButtonKey,
+  UniqueKey labelKey,
+  bool deletable,
+  TextDirection textDirection = TextDirection.ltr,
+}){
+  return _wrapForChip(
+    child: Wrap(
```

Comment:
What is the need for the `Wrap` widget? Can you remove it?

Reply:
@rami-a @HansMuller @gspencergoog 

I used a `Wrap` because some existing tests also used it, even when it has only one child.
It seems that in other tests of `chip_test.dart`, the `Chip` widget is always wrapped in some other widget, such as `Wrap`, `Column`, `Row`, or `Center`.
Does anyone know the reason behind this practice?

The ripple behaves differently when not wrapped in a `Wrap` widget (but it always makes exactly one ripple).

Reply:
One use case for the `Chip` is to be inside of a `Wrap` field that also allows entering text.  Think of the "To:" field in GMail: you type addresses, and they convert to wrapped chips.

That's why I used initially `Wrap` in tests so that we can be sure that the Chip behaves properly in that context.